cmd/root.go

```diff
@@ -48,6 +48,8 @@ func init() {
 	rootCmd.PersistentFlags().IntVar(&loggerConfig.MaxSize, "maxSize", 30, "MaxSize the max size in MB of the logfile before it's rolled")
 	rootCmd.PersistentFlags().IntVar(&loggerConfig.MaxBackups, "maxBackups", 3, "MaxBackups the max number of rolled files to keep")
 	rootCmd.PersistentFlags().IntVar(&loggerConfig.MaxAge, "maxAge", 3, "MaxAge the max age in days to keep a logfile")
+	rootCmd.PersistentFlags().String( "plugin-dir", ".", "Directory to save and load Cloudquery plugins from (env: CQ_PLUGIN_DIR)")
```

Comment:
the default value should be `.` is used only when `Getwd` is not working

```
	workingDir, err := os.Getwd()
	if err != nil {
		return "", err
	}
```

Reply:
Roger that! How does the latest commit look? I used `Getwd()` as the default and only fallback to "." if that command fails.